README.md

```
@@ -274,6 +274,112 @@ The SDK supports two main types of ads:
+    ad: ad,
+    sessionId: sessionId,
+    configuration: .default
+)
+```
+
+**Configuration Options:**
+
+The SDK provides three preset configurations for different use cases:
+
+| Configuration | Click Behavior | Use Case |
+|--------------|---------------|----------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)